package.json

```diff
@@ -7,6 +7,7 @@
     "commander": "^2.20.0",
     "dockerode": "^2.5.8",
     "execa": "^1.0.0",
+    "fs": "0.0.1-security",
```

Comment:
what's this guy? I would use builtin node `fs`, no dependency needed.

Reply:
I thought was required but you right!